frame/assets-registry/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Composable Developers"]
 edition = "2021"
 homepage = "https://composable.finance"
 name = "pallet-assets-registry"
-version = "0.1.0"
+version = "0.0.1"
```

Comment:
why are we updating this in this PR?

Reply:
Updated now , will push , I though it was the fix to some dependency issue